posts/2022-09-29-22.0.0.11-beta.adoc

```diff
@@ -0,0 +1,178 @@
+---
+layout: post
+title: "InstantOn With Checkpoint and Restore in Open liberty 22.0.0.11-beta!"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reecenana
+author_github: https://github.com/reecenana
+seo-title: InstantOn With Checkpoint and Restore in Open liberty 22.0.0.11-beta! - OpenLiberty.io
+seo-description: InstantOn will have your applications starting at light speed! Developer experience and performance enhancements come to Jakarta EE 10 for Servlet, Mail, Connectors and Persistence.
+blog_description: InstantOn will have your applications starting at light speed! Developer experience and performance enhancements come to Jakarta EE 10 for Servlet, Mail, Connectors and Persistence.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= InstantOn With Checkpoint and Restore in Open liberty 22.0.0.11-beta!
+Reece Nana <https://github.com/reecenana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+// // // // // // // //
+
+Open Liberty 22.0.0.11-beta provides an exciting new capability called InstantOn which allows applications to checkpoint the startup process and restore the application process very quickly, providing instant-on behavior for your application.
+
+Since the 22.0.0.9-beta release which included Jakarta EE 10 Core Profile, Open Liberty has continued to beta more and more Jakarta EE 10 features.  This beta release introduces various developer experience and performance enhancements available through the following new Jakarta EE 10 features: Servlet 6.0, Mail 2.1, Connectors 2.1 and Persistance 3.1.
```

Comment:
```suggestion
Since the 22.0.0.9-beta release, which included Jakarta EE 10 Core Profile, Open Liberty has continued to beta more and more Jakarta EE 10 features.  This beta release introduces various developer experience and performance enhancements available through the following new Jakarta EE 10 features: Servlet 6.0, Mail 2.1, Connectors 2.1 and Persistance 3.1.
```